hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/list/ITestS3AReadRestoredGlacierObjects.java

```diff
@@ -89,7 +92,7 @@ private FileSystem createFiles(String s3ObjectStorageClassFilter) throws Throwab
     FileSystem fs = contract.getTestFileSystem();
     Path dir = methodPath();
     fs.mkdirs(dir);
```

Comment:
skip this for a marginal increase in performance. create() assumes there's a dir and just creates the object in the destination path without checks.